core/source/org/libreoffice/ide/eclipse/core/wizards/pages/UnoPackageExportPage.java

```diff
@@ -235,12 +235,8 @@ public void modifyText(ModifyEvent pE) {
             @Override
             public void widgetSelected(SelectionEvent pE) {
                 FileDialog dlg = new FileDialog(getShell(), SWT.SAVE);
-                dlg.setFilterExtensions(new String[] { "*.oxt" });
```

Comment:
Hm maybe we want to keep this filter?

Reply:
This filter makes sure when we are viewing the contents of a folder we get to see the files ending with ".oxt". I agree with you on this that we should keep this one. 

It will be like we are making sure from our side that the user only selects ".oxt" file to overwrite not mistakenly select file of some other extension. But if he still gives some file name in the folder forcibly then our code will handle it by creating a different file with the same name but with ".oxt" extension.